PurchasesTests/Purchasing/PurchasesTests.swift

```diff
@@ -12,8 +12,10 @@ import Purchases
 class PurchasesTests: XCTestCase {
 
     override func setUp() {
-        deviceCache = MockDeviceCache()
         userDefaults = UserDefaults(suiteName: "TestDefaults")
+        userDefaults.dictionaryRepresentation().keys.forEach { userDefaults.removeObject(forKey: $0) }
+
+        deviceCache = MockDeviceCache(userDefaults: userDefaults)
```

Comment:
I think this line is enough by itself, since we're removing the persistent domain on teardown, right?

Reply:
You're 100% correct 😄